packages/flutter_tools/lib/src/commands/generate_localizations.dart

```diff
@@ -180,10 +188,16 @@ class GenerateLocalizationsCommand extends FlutterCommand {
             'Localizations.of(context), removing the need for null checking in '
             'user code.'
     );
+    argParser.addFlag(
+      'format',
+      help: 'When specified, the `dart format` command is run after generating the files.'
```

Comment:
"the files" is OK, since the reader is unlikely to think that we're referring to .arb files (or whatever), but it might be a clearer to clarify this a little.

Reply:
Fixed!